doc/api/dgram.md

```diff
@@ -454,10 +474,14 @@ s.bind(1234, () => {
 
 ### dgram.createSocket(options[, callback])
 <!-- YAML
-added: v0.11.13
+added: REPLACEME
```

Comment:
You shouldn't change the version this was added. Each function does support it's own changelog though. See the `child_process.spawn()` docs for example.